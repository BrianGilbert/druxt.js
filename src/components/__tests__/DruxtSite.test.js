import { createLocalVue, mount } from '@vue/test-utils'
import Vuex from 'vuex'

import { DruxtSite } from '../..'

// Setup local vue instance.
const localVue = createLocalVue()
localVue.use(Vuex)

describe('Druxt component', () => {
  test('defaults', () => {
    const wrapper = mount(DruxtSite, { localVue })

    // Props.
    expect(wrapper.vm.module).toBe(undefined)
    expect(wrapper.vm.propsData).toStrictEqual(undefined)
    expect(wrapper.vm.wrapper.component).toBe(undefined)
  })
})
