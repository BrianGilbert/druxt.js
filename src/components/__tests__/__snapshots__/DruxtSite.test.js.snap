// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DruxtSite component defaults 1`] = `
"<div>
  <div>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
    <druxtblockregion-stub></druxtblockregion-stub>
  </div>
</div>"
`;

exports[`DruxtSite component pending 1`] = `"<div></div>"`;
